### ISSUE TITLE
App Bundle Build Not Displaying Icons

### ISSUE THREAD

**Issue by AmyFoosh** [Gamua/Adobe-Runtime-Support#361]
Hello~

 Air Version I'm using: 33.0.2.338.

I was trying to upload an app to Google PlayStore Using the App Bundle mode, generating an AAB file for that purpose instead using 2 apks.

All worked fine, I can download it and open the app without problem, but, as you must see in this thread title, I'm not able to display app icons.

All I see is default air icon instead mine. The code line I used to create AAB file was:

```
java -jar 《path》 -package -target aab
-storetype pkcs12 -keystore file.p12 -storepass pass
app.aab app-app.xml app.swf Icons
```
I can use the same codeline replacing 《aab》 for 《apk》 and I can instead have icons on my app. That's weird~

《Icons》 on codeline is nothing but a folder with all png icons. Also, my xml file has a reference to each image inside 《Icons》 folder.

In other words, all I can think is probably aab conversion is working bad.

I read that was on an experimental basis, so I want to know if is actually a problem solved on a recent air33 version or is just a problem not resolved yet.

I also know the best way to upload an app to Google Play Store is with multiple apks, so I just need to clear my thoughts with this.

Thank you for reading and I'm very happy for you to get rid with air and not letting him die.

**Comment by ajwfrost** [comment]
This is one of the outstanding issues with the AAB format: all resources get compiled and then linked into a single resource bundle that goes into the AAB file, but where there are icons with the same name, they conflict... I think you're probably using a name like 'icon.png' but Android then picks up the file from `lib\android\lib\resources\captive_runtime\res\drawable-xhdpi` instead..

Resources - particularly if you're using ANEs - is one of the main problem areas we have with the AAB format. Will be revisiting this again at some point to see if we can find a workaround..

It might be possible to get past this if you can rename your icons to something to not clash (or maybe we can delete that AIR icon - that may just be legacy, left in the SDK but not used?!)

thanks

**Comment by AmyFoosh** [comment]
> This is one of the outstanding issues with the AAB format: all resources get compiled and then linked into a single resource bundle that goes into the AAB file, but where there are icons with the same name, they conflict... I think you're probably using a name like 'icon.png' but Android then picks up the file from `lib\android\lib\resources\captive_runtime\res\drawable-xhdpi` instead..
> 
> Resources - particularly if you're using ANEs - is one of the main problem areas we have with the AAB format. Will be revisiting this again at some point to see if we can find a workaround..
> 
> It might be possible to get past this if you can rename your icons to something to not clash (or maybe we can delete that AIR icon - that may just be legacy, left in the SDK but not used?!)
> 
> thanks

Hmmm...~ not exactly. My icons are named like "36x36.png" and so. I thought it could probably "Icon" folder, so I changed it name to "ANIC" and is still not working.

But it doesn't matter. I have no problems waiting for a solution and scratching my head trying to understand 32 and 64 bits apk upload. Hee, hee.

Anyways, I'm going to keep trying to upload it on AAB format, and if I get some progress, I'll write you back!

Thank you~

**Comment by Oldes** [comment]
Duplicate of: https://github.com/airsdk/Adobe-Runtime-Support/issues/296
